java/src/jmri/jmrit/sample/SampleToolsMenuItem.java

```diff
@@ -0,0 +1,101 @@
+package jmri.jmrit.sample;
```

Comment:
It would be best to include comments that state that the plugin should be implemented with a package that does not begin with `jmri`.

A developer _will_ generate the plugin as `jmri.jmrit.sample/SampleToolsMenuItem.java`.  Not a good thing.  

And it becomes worse when another developer does the same thing...  

And it becomes _especially bad_ when one user tries to use both of those Plugins at the same time, but only one of them "wins" (because you cannot have two different implementations using the same "package" and class name).  They expect both to work, but only one does.  And we "get" to figure out why.